internal/helpers.go

```diff
@@ -176,7 +176,10 @@ func (t *cobraTestRunner) SendText(text string) {
 	if t.stdinW == nil {
 		panic("no standard input configured")
 	}
-	t.stdinW.Write([]byte(text + "\n"))
+	_, err := t.stdinW.Write([]byte(text + "\n"))
+	if err != nil {
+		panic("Failed to to write to t.stdinW")
+	}
```

Comment:
`t` embeds `*testing.T`, so you can use require.

Reply:
That file uses panic() in 2 other places, so I'm just following how it's done here. We can update all at once later.

Reply:
Makes sense, thanks.